applications/media-player/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+    <link rel="stylesheet" href="resources/css/index.css">
+    <title>Mern MediaStream</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <div class="title"><a href="#">Mern MediaStream</a></div>
+            <div class="menu"><a href="#">ADD MEDIA</a></div>
+            <div class="menu"><a href="#">MY PROFILE</a></div>
+            <div class="menu"><a href="#">SIGN OUT</a></div>
+        </nav>        
+    </header>
+    <div class="wrapper">
+        <h2 class="gallery_title">Popular Videos</h2>
+        <div class="gallery">
+            <div class="gallery_item gallery_item1"><img src="resources/img/img1.jpg" alt=""></div>
+            <div class="gallery_item gallery_item2"><img src="resources/img/img2.jpg" alt=""></div>
+            <div class="gallery_item gallery_item3"><img src="resources/img/img3.jpg" alt=""></div>
+            <div class="gallery_item gallery_item4"><img src="resources/img/img4.jpg" alt=""></div>
+            <div class="gallery_item gallery_item5"><img src="resources/img/img5.jpg" alt=""></div>
+            <div class="gallery_item gallery_item6"><img src="resources/img/img6.jpg" alt=""></div>
+            <div class="gallery_item gallery_item7"><img src="resources/img/img7.jpg" alt=""></div>
+            <div class="gallery_item gallery_item8"><img src="resources/img/img8.jpg" alt=""></div>
+            <div class="gallery_item gallery_item9"><img src="resources/img/img9.jpg" alt=""></div>            
```

Comment:
This should be in an `<ul>`.

Also, why create a `class` for each `<div>` and then not use them in the CSS?